### PR TITLE
Change examples under the documentation annotation in more descriptive manner. 

### PR DESCRIPTION
**Pull request by ChathurikaA**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java [hunk 1]
```diff
@@ -360,15 +382,14 @@ public void connect() throws ConnectionUnavailableException {
             if (e.getCause() instanceof MailConnectException) {
                 if (e.getCause().getCause() instanceof ConnectException) {
                     throw new ConnectionUnavailableException("Error is encountered while connecting the smtp"
-                            + " server by the email ClientConnector with properties: "
-                            + initProperties.toString(), e);
+                            + " server" +  e.getMessage(), e.getCause());
                 } else {
-                    throw new RuntimeException("Error is encountered while connecting  to the server "
-                            + "with properties: " + initProperties.toString() , e);
+                    throw new RuntimeException("Error is encountered while connecting to the smtp server" +
```
Comment (tishan89) [inline component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java]: Add full-stop and a space to end of line

File: component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java [hunk 2]
```diff
@@ -360,15 +382,14 @@ public void connect() throws ConnectionUnavailableException {
...
+                            e.getMessage(), e.getCause());
                 }
             } else {
                 throw new RuntimeException("Error is encountered while connecting to the"
-                        + " server with properties: " + initProperties.toString() , e);
+                        + " the smtp server" + e.getMessage(), e);
```
Comment (tishan89) [inline component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java]: There is a repeating the and missing full-stop and space

**Review by tishan89 — CHANGES_REQUESTED** [review]

File: component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java (hunk 1 above)
Comment (ChathurikaA) [reply component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java]: Fix in #8b4b372

File: component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java (hunk 2 above)
Comment (ChathurikaA) [reply component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java]: Fix in #8b4b372

**Review by tishan89 — APPROVED** [review]

**Pull request merged**